pylint_odoo/checkers/itp_checks.py

```diff
@@ -141,7 +141,8 @@ def _check_manifest_image(self):
             images = manifest_dict.get('images', [])
             if images != []:
                 for image in images:
-                    if image.startswith("images/"):
+                    if "images/" in image:
```

Comment:
Why did you replaced it?

Reply:
I think it's correct too and it's short

Reply:
It's not the same. It allows file name static/images/1.png, which is not correct